src/main/java/study/lotto/view/dto/RequestMoney.java

```diff
@@ -46,4 +46,8 @@ public int hashCode() {
     public String toString() {
         return String.valueOf(money);
     }
+
+    public double winningRate(long sum) {
+        return (1.0 * sum) / money;
```

Comment:
1.0을 곱하는건 형변환떄문에 하신건가요? 아래와 같이 표기하는게 어떨까요?
```suggestion
        return (double) sum / money;
```

Reply:
getMoney() 메서드는 더 이상 사용되는 곳이 없는 것 같네요. 삭제하는게 좋겠습니다. 🙂